Tests/InstallationTests/ReceiptParserInstallation/ReceiptParserIntegrationTests/ReceiptParserIntegrationTests.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  ReceiptParserIntegrationTests.swift
+//  ReceiptParserIntegrationTests
+//
+//  Created by Nacho Soto on 11/29/22.
+//
+
+import ReceiptParser
+import ReceiptParserIntegration
+import XCTest
+
+final class ReceiptParserIntegrationTests: XCTestCase {
+
+    func testCanParseReceipts() throws {
```

Comment:
Maybe rename to `testParsingEmptyReceiptThrowsError` or something like that? In case we add more tests in the future.

Reply:
Yeah good point.